types/shexj/index.d.ts

```diff
@@ -0,0 +1,576 @@
+// Type definitions for ShExJ 2.1
```

Comment:
npmjs does not show any package with this name. so this needs to be non npm package

Reply:
There's a constellation of ShEx packages in the `@shexjs` scope. They provide code to manipulate ShExJ structures. These structures were [formally modelled](http://shex.io/shex-semantics/#shexj) but never before in Typescript. There are some other packages which use ShEx as well. The idea is that `@types/shexj` be a standard structure all (ts-aware) ShExJ tools can share.

I think the closest analogy would be `@types/rdf-js` which defines structures and interfaces used by e.g. `n3`, `graphy` etc. Likewise, `@shexjs/core`, `@shexjs/parser`, and a lot of pending code in [solid](https://github.com/solid/solid) will leverage this typing.

Given that, is it necessary to overlap with an existing npm name?

Reply:
For the `rdf-js` typings, it was [suggested](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20507#issuecomment-336071454) that the npm package would also be claimed (which I did). Just to make sure that the TS tooling would not break. So I think it's a good idea to claim `shexjs` just like [`rdf-js`](https://www.npmjs.com/package/rdf-js).

Reply:
I think I've take ownership two lines below per [Definition owners](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#definition-owners).
Is some other speech act required?

Reply:
If you remove the rule linter will show you how to fix that error by updating header comment

Reply:
gotcha! ok, fixed.

The linter was somewhat helpful in that it suggested adding:

> // Type definitions for non-npm package shexj-browser

In reality, I had to replace the existing header line with a header with "non-npm package" and a version:

> // Type definitions for non-npm package ShExJ 2.1